src/views/AfhandelingView.vue

```diff
@@ -583,6 +584,11 @@ const saveVraag = async (vraag: Vraag, gespreksId?: string) => {
   }
 
   const savedContactmoment = await saveContactmoment(contactmoment);
+  await upsertContactmomentManagementInfo(
```

Comment:
heb je overwogen om dit op de server te doen, net als de verwerkingslog?

Reply:
Zeker, en dat kunnen we wmb alsnog doen. Ik was alleen parallel aan de contactmomenten stories begonnen, en in die fase leek  het me het qua afhankelijkheden handiger om met een los endpoint te beginnen